### ISSUE TITLE
API Export Error

### ISSUE THREAD

**Issue by kennywasafk** [5hay/notionbackup#1]
This was working brilliantly for me for a while, and then, after I'm presuming a change in Notion's API, I've started getting the error from the log below. Would be fantastic if you could take a look. Thanks in advance!

```
2022/04/27 20:12:16 notionbackup (v1.0.0) | Starting the export process ...
2022/04/27 20:17:56 http.Post('https://www.notion.so/api/v3/getTasks') returned non-200 status code of 502
```

**Comment by 5hay** [comment]
Hey, thanks for letting me know. I will take a look in the next couple of days and give an update here.

**Comment by 5hay** [comment]
I just did multiple exports and every one worked perfectly. I'm using the same version. 
Maybe the API just had a hiccup or something when you were trying to use it? Also maybe try to get a new token by logging out and in again.

My automated weekly backup also just ran fine.

**Comment by kennywasafk** [comment]
Re-logging and using a new token_v2 got it to work, thank you! Sorry to have wasted your time on this. Although maybe there's some sense in having a bad token come back with a specific descriptive error, for user experience purposes rather then anything else? Just a thought, in case this kind of feedback is useful.

While I have you though, I'm not sure if this is intended but while backing up specific pages works fine, backing up the whole workspace seems to time out. I'm using the Workspace ID found under Settings & Members > Security & identity > Setup Information when specifying NOTION_PAGEID, and getting the below.

```
2022/04/30 14:09:48 notionbackup (v1.0.0) | Starting the export process ...
2022/04/30 15:25:04 Post "https://www.notion.so/api/v3/getTasks": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
```

**Comment by 5hay** [comment]
> Although maybe there's some sense in having a bad token come back with a specific descriptive error, for user experience purposes rather then anything else? Just a thought, in case this kind of feedback is useful.

Definitely, thanks for that. I need to test further, but I don't think an invalid token would result in a 502. At least I hope it's not the case.

> While I have you though, I'm not sure if this is intended but while backing up specific pages works fine, backing up the whole workspace seems to time out. I'm using the Workspace ID found under Settings & Members > Security & identity > Setup Information when specifying NOTION_PAGEID, and getting the below.

Yeah, it doesn't work like that. The tool requires to have a Page ID and that's different from a Workspace ID.
I need to update the blog or at least the README, but here's a tweet where I explained how to back up the whole workspace: 

https://twitter.com/5hay/status/1405041000389697538?s=20&t=d7W6GoUIYubNlPVizWHOgQ

TL;DR: Just create a root page in Notion and move everything else inside of it. Then state the Page ID of this root page as the NOTION_PAGEID.

Feel free to ask more, happy to help.

**Comment by kennywasafk** [comment]
Ah, I see, thank you for the explanation makes complete sense. Nothing else to ask, everything else is pretty straight forward! Thank you for your help again 👍
